app/views/index.erb

```diff
@@ -0,0 +1,37 @@
+<div>
+	<div id = "glitter">
+		<h1>Chitter</h1>
+	</div>
+</div>
+
+<% if current_user %>
+	<p>
+		What do you want to say to the world that is so urgent?
+	</p>
+	<form action='/new_peep' method='post' id = 'new_peep'>
+		<label for='title'>
+			Title: <input name='title' type='text'>
+		</label>
+		<label for='content'>
+			All this * by you: <input name='content' type='text'>
+		</label>
+		<button>Create a new Peep</button>
+	</form>
+<% else %>
+	<p>
+		Log in to post peeps
+	</p>
+<% end %>
+
+<% if !!@peeps %>
```

Comment:
this is redundant as `!!@peeps` will always be true, since `@peeps` will always be something (even if it's empty)